bin/zerobufCxx.py

```diff
@@ -775,11 +798,61 @@ def __init__(self, item):
         self.type = item[2]
         self.values = item[3:]
```

Comment:
Can you change the function definition to take 3 parameters and do the unpacking in the caller?